### DISCUSSION TITLE
Can MOOSE compute local volume of Element?

### DISCUSSION THREAD

**Discussion by hokonkim** [idaholab/moose#19305] [Q&A General] [answered]
Hello, I have a question about computing local volume of Element in MOOSE.

If possible, I'd like to compute local volume of Element and store the value.

Are there proper ways to do this or any tips?

I appreciate your help.

Sincerely,
Hokon Kim

**Comment by loganharbour** [comment]
There's a few ways to do this - but funny enough, I have a need for this capability at the very moment. I'll put up a PR shortly with the object.

**Comment by dschwen** [comment]
It is provided as `_current_elem_volume`

**Comment by loganharbour** [comment, marked as answer]
See #19308 for the new object.

**Reply by hokonkim** [reply]
As far as I understand, "VolumeAux" is added (available) in the latest MOOSE framework.

To use this latest object, I need to update the MOOSE, right?
(I'm using MOOSE on HPC, so I need a help from technical support team for update)

**Reply by GiudGiud** [reply]
Yes you need to update MOOSE to have access to it. It's in since Saturday on the master branch
